akka-actor/src/main/scala/akka/dispatch/sysmsg/SystemMessage.scala

```diff
@@ -237,12 +237,12 @@ private[akka] final case class Supervise(child: ActorRef, async: Boolean) extend
  * INTERNAL API
  */
 @SerialVersionUID(1L)
-private[akka] case class Watch(watchee: InternalActorRef, watcher: InternalActorRef) extends SystemMessage // sent to establish a DeathWatch
+private[akka] final case class Watch(watchee: InternalActorRef, watcher: InternalActorRef) extends SystemMessage // sent to establish a DeathWatch
 /**
  * INTERNAL API
  */
 @SerialVersionUID(1L)
-private[akka] final case class Unwatch(watchee: ActorRef, watcher: ActorRef) extends SystemMessage // sent to tear down a DeathWatch
+private[akka] final case class Unwatch(watchee: InternalActorRef, watcher: InternalActorRef) extends SystemMessage // sent to tear down a DeathWatch
```

Comment:
Does deserialization of the old with the new still work (and vice versa)? If not then we need to roll this back.